common/src/main/java/tds/common/web/interceptors/EventLoggerInterceptor.java

```diff
@@ -26,7 +26,8 @@
 
     private final String app;
     private final ObjectMapper objectMapper;
-    private EventLogger eventLogger;
+    private final String EVENT_LOGGER_ATTRIBUTE = "event-logger";
```

Comment:
Can you explain what the problem was and why this fixes it?  I don't have the context to know why this fixes the concurrent issue.